internal/auth/jwt.go

```
@@ -0,0 +1,90 @@
+
+			if audience := r.Header.Get(HeaderJWTAudience); audience != "" {
+				claimAud, ok := claims["aud"].(string)
+				if !ok || claimAud != audience {
+					jsonError(w, "Invalid audience", http.StatusUnauthorized)
+					return
+				}
+			}
+
+			for key, value := range claims {
+				if str, ok := value.(string); ok {
+					headerKey := fmt.Sprintf("%s%s", HeaderPrefix, strings.Title(key))
```

golangci-lint (1.62.2)

SA1019: strings.Title has been deprecated since Go 1.18 and an alternative has been available since Go 1.0: The rule Title uses for word boundaries does not handle Unicode punctuation properly. Use golang.org/x/text/cases instead. 

(staticcheck)